api/src/client.rs

```diff
@@ -81,6 +81,18 @@ impl MLBApi {
         self.get(url).await
     }
 
+    pub async fn get_standings_on_date<T: Datelike>(&self, date: T) -> StandingsResponse {
```

Comment:
since you've added to the client here, can you bump the version in `api/Cargo.toml` to 0.0.10, and then update the corresponding package in the root `Cargo.toml`